src/FSharpLint.Core/Text.resx

```diff
@@ -174,6 +174,9 @@
   <data name="RulesNamingConventionsUnderscoreError" xml:space="preserve">
     <value>Consider changing `{0}` to remove any underscores.</value>
   </data>
+  <data name="RulesVariableLengthGreaterThanOneError" xml:space="preserve">
+    <value>Consider changing `{0}` to have minimum length of two.</value>
```

Comment:
let's not mention two, just say more than one

Reply:
Updated.